### DISCUSSION TITLE
[BUG] alist本地储存出现问题（不确定是不是alist问题）

### DISCUSSION THREAD

**Discussion by LINKlang** [AlistGo/alist#5307] [Q&A] [answered]
本地储存上传大文件会提示
**Cannot read properties of undefined (reading 'status')**

小文件（似乎？）没有出现

使用了cf代理域名，但没有用cdn
因为我只用本地储存，所以没有试过网盘

搭建位置：甲骨文云1h1g大版vps

控制台提示(打码了域名)
`{
    "message": "Network Error",
    "name": "AxiosError",
    "stack": "AxiosError: Network Error\n    at c.onerror (http://***.***.***/assets/index.2ea96c44.js:21:4558)",
    "config": {
        "transitional": {
            "silentJSONParsing": true,
            "forcedJSONParsing": true,
            "clarifyTimeoutError": false
        },
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1,
        "env": {},
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IlJvYm90MDQyIiwiZXhwIjoxNjk2MTQ3NDIzLCJuYmYiOjE2OTU5NzQ2MjMsImlhdCI6MTY5NTk3NDYyM30.FZgDclSKSmdPdScTE6ww0HxJ_6BnjXcycOC6kZq-ncY",
            "Content-Type": "application/octet-stream",
            "File-Path": "%2F%E6%9C%AC%E6%9C%BA%E5%82%A8%E5%AD%98%2Fcom.kiloo.subwaysurf.cn_4.07.1_und3fined.ipa",
            "Last-Modified": "1695974668539",
            "Password": ""
        },
        "baseURL": "http://***.***.***/api",
        "withCredentials": false,
        "method": "put",
        "url": "/fs/put",
        "data": {}
    },
    "code": "ERR_NETWORK",
    "status": null
}`
还有
`Upload.b75efd63.js:1  TypeError: Cannot read properties of undefined (reading 'status')
    at index.2ea96c44.js:21:14318
    at async Object.fe [as upload] (Upload.b75efd63.js:1:1823)
    at async W (Upload.b75efd63.js:1:3649)
    at async Upload.b75efd63.js:1:477
    at async c (Upload.b75efd63.js:1:398)
    at async de (Upload.b75efd63.js:1:565)
    at async $ (Upload.b75efd63.js:1:3414)
W @ Upload.b75efd63.js:1
await in W(异步)
(anonymous) @ Upload.b75efd63.js:1
de @ Upload.b75efd63.js:1
$ @ Upload.b75efd63.js:1
onChange @ Upload.b75efd63.js:1
`
和
`index.2ea96c44.js:21 
PUT http://***.***.***/api/fs/put net::ERR_CONNECTION_RESET
(anonymous) @ index.2ea96c44.js:21
zl @ index.2ea96c44.js:21
jl @ index.2ea96c44.js:21
Promise.then(异步)
request @ index.2ea96c44.js:21
(anonymous) @ index.2ea96c44.js:21
(anonymous) @ index.2ea96c44.js:20
fe @ Upload.b75efd63.js:1
W @ Upload.b75efd63.js:1
(anonymous) @ Upload.b75efd63.js:1
de @ Upload.b75efd63.js:1
$ @ Upload.b75efd63.js:1
onChange @ Upload.b75efd63.js:1
`

**Comment by meidaim** [comment]
【求解】我也出现了同样的问题，小文件没事。大文件就是报错
本地储存上传大文件会报错 提示 
Cannot read properties of undefined (reading 'status')
![报错提示](https://github.com/alist-org/alist/assets/130755827/a3f28908-6121-4fb8-a178-a845893c9909)

**小文件（似乎？）没有出现**
我也使用了cf代理域名，但没有用cdn 这是啥情况啊？【求解】

**Reply by LINKlang** [reply]
看下是不是用了nginx

**Comment by Deepseaon** [comment, marked as answer]
nginx反代的话把bodysize设大点，然后我自己是开cdn的，开cdn的话无论如何都不能太大，因为cf免费版好像限制最大几个g来着，只能换ip访问上传（不过10来g的文件也还是有这个问题，还没解决）

**Reply by LINKlang** [reply]
我已经解决了，就是bodysize开大，然后给cdn关了
